### PR TITLE
Clarify the TraceIdRatioBased description format

### LINKED ISSUE #1524: Clarify TraceIdRatioBased Sampler description
[From the specification](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.0.1/specification/trace/sdk.md#traceidratiobased):

> Description MUST be `TraceIdRatioBased{0.000100}`

Does this mean this sampler should always return the static string `"TraceIdRatioBased{0.000100}"` even if the ratio is not `0.000100`?

Or should it return `"TraceIdRatioBased{$RATIO}"` with `$RATIO` replaced with a decimal representation of the sampling ratio? If so, what precision should this ratio be?

### PR DESCRIPTION
**Pull request by MrAlias**
Samplers need to return a static string as a description. The string the TraceIdRatioBased sampler returns needs to be based on the sampling ratio that it is configured for. The current specification could be misinterpreted as requiring a static string that was not based on the samplers sampling ratio be used. This corrects that discrepancy.

Resolves #1524

### CODE REVIEW

**Review by reyang — APPROVED** [review]
Good catch!

File: specification/trace/sdk.md [hunk 1]
```diff
@@ -252,7 +252,13 @@ The default sampler is `ParentBased(root=AlwaysOn)`.
 * The `TraceIdRatioBased` MUST ignore the parent `SampledFlag`. To respect the
 parent `SampledFlag`, the `TraceIdRatioBased` should be used as a delegate of
 the `ParentBased` sampler specified below.
-* Description MUST be `TraceIdRatioBased{0.000100}`.
+* Description MUST return a string of the form `"TraceIdRatioBased{RATIO}"`
+  with `RATIO` replaced with the Sampler instances' trace sampling ratio
```
Comment (arminru) [inline specification/trace/sdk.md:256]:
```suggestion
  with `RATIO` replaced with the Sampler instance's trace sampling ratio
```
It's for _one_ instance, right?

**Review by arminru — APPROVED** [review]
It's just a clarification but I think we should still mention it in the changelog.

**Review by carlosalberto — APPROVED** [review]

**Comment by carlosalberto** [comment]
(Probably worth mentioning this in the maintainers meeting, for everybody is aware of this detail ;) )

File: specification/trace/sdk.md [hunk 2]
```diff
@@ -252,7 +252,13 @@ The default sampler is `ParentBased(root=AlwaysOn)`.
 * The `TraceIdRatioBased` MUST ignore the parent `SampledFlag`. To respect the
 parent `SampledFlag`, the `TraceIdRatioBased` should be used as a delegate of
 the `ParentBased` sampler specified below.
-* Description MUST be `TraceIdRatioBased{0.000100}`.
+* Description MUST return a string of the form `"TraceIdRatioBased{RATIO}"`
+  with `RATIO` replaced with the Sampler instance's trace sampling ratio
+  represented as a decimal number. The precision of the number SHOULD follow
+  implementation language standards and SHOULD be high enough to identify when
+  Samplers have different ratios. For example, if a TraceIdRatioBased Sampler
+  had a sampling ratio of 1 to every 10,000 spans it could return
```
Comment (iNikem) [inline specification/trace/sdk.md:260]:
```suggestion
  had a sampling ratio of 1 to every 10,000 spans it COULD return
```

**Review by iNikem — APPROVED** [review]

**Pull request merged**